qemu/tests/smbios_default_check.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import re
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check default smbios strings in qemu :
+    1) Boot guest with default smbios set up
+    2) Verify if bios info have been emulated correctly
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def check_info(cmd, template):
+        msg_log = "Check " + template + " info"
+        error_context.context(msg_log, logging.info)
+        cmd_output = session.cmd_output(cmd)
+        cmd_output_re = re.split('\n', cmd_output.strip('\n'))[-1].strip(' ')
+        template = params.get(template)
+        return re.match(template, cmd_output_re)
+
+    re_template = []
+    re_template.append("System_Manufacturer")
+    re_template.append("System_SKU_Number")
+    re_template.append("Baseboard_Manufacturer")
+    re_template.append("Baseboard_Product_Name")
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = int(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=timeout)
+
+    failures = []
+    check_info_cmd = []
+    check_info_cmd.append(params.get("get_sys_manufacturer"))
+    check_info_cmd.append(params.get("get_sys_SKUNumber"))
+    check_info_cmd.append(params.get("get_baseboard_manufacturer"))
+    check_info_cmd.append(params.get("get_baseboard_product_name"))
```

Comment:
It's better to use "params['get_baseboard_product_name']".
Line 24, Line 40-42 are the same.

Reply:
Your suggestion is make sense,I had rewrited the code.Thanks!